src/test/java/webdriver/DriverFactory.java

```diff
@@ -0,0 +1,35 @@
+package webdriver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+
+import java.util.concurrent.TimeUnit;
+
+public class DriverFactory {
+    private static final String CHROMEDRIVER_EXE_PATH = "src/main/resources/chromedriver.exe";
+    public static final String ONLINER_URL = "https://www.onliner.by/";
+    public static final int IMPLICITLY_WAIT = 10;
+
+    public enum BrowserType {FIREFOX, CHROME, EDGE, IE, OPERA}
+
+    public static WebDriver getDriver(BrowserType type) {
+        WebDriver driver = null;
+        switch (type) {
+            case CHROME:
+                driver = getChromeDriver();
+                break;
+        }
+        assert driver != null;
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(IMPLICITLY_WAIT, TimeUnit.SECONDS);
+        driver.get(ONLINER_URL);
```

Comment:
Please keep your methods SOLID. One method - one responsibility, here in this method you are configuring driver AND navigating to url. 
Anyway is this the right place to navigate to url?